curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9343769e8f85c1e17e05.md

```diff
@@ -14,13 +14,17 @@ Create a `div` element with the class of `task`. Utilize template strings to set
 You should create a `div` element with the class `task`.
 
 ```js
-assert.match(code, /<div\s+class\s*=\s*('|")task\1/)
+const classFirst = `<div\s+class\s*=\s*('|")task\1`;
+const idFirst = `<div\s+id\s*=\s*('|")task\2\s+class\s*=\s*('|")task\3`
+assert.match(code, new Regex(`${classFirst}|${idFirst}`));
```

Comment:
this fails with `Uncaught SyntaxError: Octal escape sequences are not allowed in template strings.`

you will need to check if there are more escapes needed
also you still have id=task